stable/prometheus-operator/values.yaml

```diff
@@ -964,6 +964,9 @@ prometheusOperator:
       podAnnotations: {}
       nodeSelector: {}
 
+  ## Namespaces not to scope the interaction of the Prometheus Operator (deny list).
```

Comment:
Could you figure out why circle-ci chart lint report this error please?

stable/prometheus-operator/values.yaml
  967:5     error    missing starting space in comment  (comments)

Reply:
Should be better now.